eth_abi/codec.py

```diff
@@ -100,7 +112,20 @@ def is_encodable(self, typ: TypeStr, arg: Any) -> bool:
 
 
 class ABICodec(BaseABICodecEncoder):
+    """
+    Codec used to encode and decode values.
+    """
+
     def __init__(self, registry: ABIRegistry=None):
+        """
```

Comment:
It looks like we can get the `__init__` docs added to the class docs by adding `autoclass_content = 'both'` to the docs conf. Can you experiment with it to see if that works as expected, looks okay, etc?  https://stackoverflow.com/a/9772922/8412986

If so, we will probably add it to our project template so that it eventually gets filters to all our other projects.